common/names.go

```diff
@@ -10,7 +10,6 @@ import (
 
 const (
 	defaultDiskMakerImageVersion = "quay.io/openshift/origin-local-storage-diskmaker"
-	defaultProvisionImage        = "quay.io/openshift/origin-local-storage-static-provisioner"
```

Comment:
We also have references to this image in some other places as well.  Like  `config/manager/manager.yaml` , `config/manifest/4.9/image-references` and in the csv files.

Reply:
Thank @sp98, updated.